inspirehep/modules/theme/templates/inspirehep_theme/format/record/Experiment_HTML_detailed.tpl

```diff
@@ -23,126 +25,151 @@
 {% extends "inspirehep_theme/format/record/Inspire_Default_HTML_detailed.tpl" %}
 
 {% block body %}
-  <div class="row">
-    <div class="col-md-12">
-      <h3>
-      {% for collection in record['collections'] %}
-        {% if 'primary' in collection %}
-          <span class="label label-default pull-left">
-          {{ collection['primary'] }}</span>
-        {% endif %}
-      {% endfor %}
-      </h3>
-    </div>
-  </div>
-  <div class="row"><div class="col-md-12"><p></p></div></div>
-  <div class="row"><div class="col-md-12"><p></p></div></div>
-  <div class="row">
-    <div class="col-md-12">
-     <div class="record-title pull-left">
-        {{ record['experiment_names'][0] }}
-        {% if record['affiliation'] %}
-          ({{ record['affiliation'][0] }})
+<div id="record_content">
+  <div class="record-detailed record-detailed-experiments">
+    <div class="panel">
+      <div class="panel-heading">
+        <h1 class="record-detailed-title">
+          {{ record['experiment_names'][0].title }}
+            {% if record['affiliation'] %}
+              ({{ record['affiliation'][0] }})
+            {% endif %}
+        </h1>
+        {% if record['titles'] %}
+          <h2 class="record-detailed-subtitle record-detailed-subtitle-experiments">{{ record['titles'][0].title }}</h2>
         {% endif %}
       </div>
+      <div class="panel-body">
+        <div class="row">
+          <div class="col-md-12">
+
+            {% if record['urls'] %}
+            <div class="detailed-record-field">
+              {% for url in record['urls'] %}
+                <a href="{{ url['value'] }}">{{ url['value'] }}</a><br>
+              {% endfor %}
+            </div>
+            {% endif %}
+
+            {% if record['date_started'] %}
+              <div class="detailed-record-field">
+                <label>Dates:</label>
+                {{ record|experiment_date }}
+                <br></div>
+            {% endif %}
+
+            {% if record['spokespersons'] %}
+              <div class="detailed-record-field">
+                <label>Spokesperson:</label>
+                {% for spokesperson in record['spokespersons'] %}
+                {{ spokesperson.name }}{% if not loop.last %}; {% endif %}
+              {% endfor %}
+              </div>
+              {% endif %}
```

Comment:
Oh wait. It's actually spelled "spokesperson"! TIL: https://en.wikipedia.org/wiki/Spokesperson